src/linear_algebra/matrix/pos_def.lean

```diff
@@ -56,6 +56,26 @@ begin
   apply hM.2 x hx,
 end
 
+namespace pos_def
+
+variables {M : matrix n n ℝ} (hM : M.pos_def)
+include hM
+
+lemma pos_det [decidable_eq n] : 0 < det M :=
```

Comment:
I think we generally write `pos` after the thing that is positive, so:
```suggestion
lemma det_pos [decidable_eq n] : 0 < det M :=
```
(Although I do appreciate the nicer rhythm of `pos_def.pos_det`.)